google-map.html

```diff
@@ -34,7 +34,62 @@
 @status alpha
 @homepage http://googlewebcomponents.github.io/google-map
 -->
-<polymer-element name="google-map-marker" attributes="icon">
+<!--
+Fired when the marker icon was clicked.
+
+@event google-map-marker-click
+@param {google.maps.MouseEvent} event The mouse event.
+-->
+
+<!--
+Fired when the marker icon was double clicked.
+
+@event google-map-marker-dblclick
+@param {google.maps.MouseEvent} event The mouse event.
+-->
+
+<!--
+Fired for a mousedown on the marker.
+
+@event google-map-marker-mousedown
+@param {google.maps.MouseEvent} event The mouse event.
+-->
+
+<!--
+Fired when the DOM `mousemove` event is fired on the marker.
+
+@event google-map-marker-mousemove
+@param {google.maps.MouseEvent} event The mouse event.
+-->
+
+<!--
+Fired  when the mouse leaves the area of the marker icon.
+
+@event google-map-marker-mouseout
+@param {google.maps.MouseEvent} event The mouse event.
+-->
+
+<!--
+Fired when the mouse enters the area of the marker icon.
+
+@event google-map-marker-mouseover
+@param {google.maps.MouseEvent} event The mouse event.
+-->
+
+<!--
+Fired for a mouseup on the marker.
+
+@event google-map-marker-mouseup
+@param {google.maps.MouseEvent} event The mouse event.
+-->
+
+<!--
+Fired for a rightclick on the marker.
```

Comment:
Can you also add to these descriptions that these events are only fired if clickEvents/mouseEvents are set and refer people to those properties.